contrib/completions/bash/kubectl

```diff
@@ -2302,6 +2358,18 @@ _kubectl_annotate()
 
     must_have_one_flag=()
     must_have_one_noun=()
+    must_have_one_noun+=("componentstatuses")
+    must_have_one_noun+=("events")
+    must_have_one_noun+=("horizontalpodautoscalers")
+    must_have_one_noun+=("limitranges")
+    must_have_one_noun+=("nodes")
+    must_have_one_noun+=("persistentvolumeclaims")
+    must_have_one_noun+=("persistentvolumes")
+    must_have_one_noun+=("pods")
+    must_have_one_noun+=("replicationcontrollers")
```

Comment:
Background: there is an `argAliases` field no in the Command, next to `validArgs`. That's meant to include the plural while the `validArgs` are singular.